package.json

```diff
@@ -208,49 +204,33 @@
     "jest-environment-jsdom": "^28.1.3",
     "lint-staged": "^15.2.1",
     "moment-timezone": "^0.5.43",
-    "postcss": "^8.1.10",
-    "postcss-loader": "^4.1.0",
     "prettier": "^2.2.0",
     "react": "^18.1.0",
     "react-dom": "^18.1.0",
     "react-test-renderer": "^18.1.0",
-    "rollup": "^2.79.1",
-    "rollup-plugin-copy": "^3.4.0",
-    "rollup-plugin-insert": "^1.3.1",
-    "rollup-plugin-node-globals": "^1.4.0",
-    "rollup-plugin-peer-deps-external": "^2.2.4",
-    "rollup-plugin-terser": "^7.0.2",
-    "rollup-plugin-url": "^3.0.1",
-    "rollup-plugin-visualizer": "^4.2.0",
     "semantic-release": "^19.0.5",
     "stream-chat": "^8.21.0",
     "style-loader": "^2.0.0",
     "ts-jest": "^28.0.8",
-    "typescript": "^4.7.4",
-    "url-loader": "^4.1.1",
-    "webpack": "4.44.2",
-    "webpack-cli": "^3.3.12",
-    "webpack-dev-server": "~3.11.0"
+    "typescript": "^5.4.2",
+    "url-loader": "^4.1.1"
   },
   "scripts": {
-    "analyze": "yarn build --stats && webpack-bundle-analyzer build/bundle-stats.json",
-    "build": "rm -rf dist && mkdir -p dist/assets && yarn --silent copy-version && yarn build-translations && yarn types && yarn bundle",
-    "bundle": "rollup -c",
-    "bundle-size": "BUNDLE_SIZE=true yarn bundle",
+    "build": "rm -rf dist && ./scripts/copy-version.sh && yarn build-translations && yarn bundle",
+    "bundle": "concurrently tsc ./scripts/copy-css.sh ./scripts/bundle.mjs",
     "build-translations": "i18next",
-    "copy-version": "echo '\u001b[34mℹ\u001b[0m Copying Version to \u001b[34msrc/version.ts\u001b[0m' && PACKAGE_VERSION=$(node -pe 'require(`./package.json`).version') && PACKAGE_STRING=\"'$PACKAGE_VERSION'\" && echo 'export const version = '$PACKAGE_STRING';' > ./src/version.ts && echo '\u001b[32m✓\u001b[0m Done Copying Version'",
     "coverage": "jest --collectCoverage && codecov",
     "eslint": "eslint '**/*.{js,md,ts,jsx,tsx}' --max-warnings 0",
     "lint": "prettier --list-different 'src/**/*.{js,ts,tsx,md,json}' .eslintrc.json .prettierrc babel.config.js && eslint 'src/**/*.{js,ts,tsx,md}' --max-warnings 0 && yarn validate-translations",
     "lint-fix": "prettier --write 'src/**/*.{js,ts,tsx,md,json}' .eslintrc.json .prettierrc babel.config.js && eslint --fix 'src/**/*.{js,ts,tsx,md}' --max-warnings 0",
     "prettier": "prettier --list-different '**/*.{js,ts,tsx,md,json}' .eslintrc.json .prettierrc babel.config.js",
     "prettier-fix": "prettier --write '**/*.{js,ts,tsx,md,json}' .eslintrc.json .prettierrc babel.config.js",
     "fix-staged": "lint-staged --config .lintstagedrc.fix.json --concurrent 1",
-    "start": "tsc --watch",
-    "prepare": "husky install && yarn run build",
+    "start": "tsc --watch --declarationMap",
```

Comment:
In "dev mode" it's nice to have declaration maps, so that clicking on types in example apps takes you straight to the source TS file (instead of `.d.ts` from dist).